src/entity.ts

```diff
@@ -696,6 +697,55 @@ export namespace entity {
     });
   }
 
+  /**
+   * Find the properties which value size is large than 1500 bytes,
+   * with excludeLargeProperties enabled, automatically exclude properties from indexing.
+   * This will allow storing string values larger than 1500 bytes
+   *
+   * @param entities Datastore key object(s).
+   * @param path namespace of provided entity properties
+   * @param properties properties which value size is large than 1500 bytes
+   */
+  export function findLargeProperties_(
+    entities: Entities,
+    path: string,
+    properties: string[] = []
+  ) {
+    const MAX_DATASTORE_VALUE_LENGTH = 1500;
+    if (Array.isArray(entities)) {
+      for (const entry of entities) {
+        if (entry.hasOwnProperty('name') && entry.hasOwnProperty('value')) {
+          if (
+            typeof entry.value === 'string' &&
```

Comment:
Since we have it, let's use `is.string(entry.value)`

Reply:
done